testsuite/classlibrary/TestUGen_Duty.sc

```diff
@@ -0,0 +1,48 @@
+TestUGen_Duty : UnitTest {
```

Comment:
just looking at https://github.com/supercollider/supercollider/tree/master/testsuite/classlibrary:

we have `TestFilterUGens`, `TestCoreUGens`, `TestPV_ChainUGen`, and `TestSanitize`. we should come up with a consistent naming scheme. i'll open a separate issue for it.

Reply:
OK, that would be good. I moved "UGen" forward to make it a little more like a namespace.